README.adoc

```diff
@@ -1,48 +1,166 @@
+- Extract and Navigate to `installer` directory
+- Now execute the file `install.cmd` (You can simply double click)
+- As the app is unrecognized, Windows may launch up a new Dialog titled as `Windows protected your PC` with description `Microsoft Defender SmartScreen prevented an unrecognised app from starting. Running this app might put your PC at risk.`
+- You may click on `More info` and the choose `Run anyway`
+- You will then be **prompted** with a series of **dialog boxes** asking **permission** for the operatons(modifying the registry). Please **Select Yes** in all those prompts to allow the installation.
+- Test your installation by typing `starfix` on the Terminal(Command Prompt or Windows PowerShell)
+- To set-up the Browser Extension navigate to this section: https://github.com/fahad-israr/starfish/blob/doc/README.adoc#browser-extension[Broswer Extension]
+
+=== MACOS
+- Download the `starfix-macos.zip` file from the Latest Realease
+- Extract and Navigate to `installer` directory
+- On the terminal execute the following(ensure the current directory is `installer` ) 
+
+[source, bash]
+----
+./install
+----
+
+- Once you execute the above command you'll  be prompted for password (for sudo )
+- Test your installation by going to terminal and typing `starfix`
+- To set-up the Browser Extension navigate to this section: https://github.com/fahad-israr/starfish/blob/doc/README.adoc#browser-extension[Broswer Extension]
+
+=== BROWSER EXTENSION
+-  Clone This Repository.
+- Now **Load Unpacked Extension** in the `browser-extension` directory.The method for Loading Unpacked extension generally varies  across **browsers** . We'll guide you with loading up on **Google Chrome** and **Mozilla Firefox**.
+
+==== Loading unpacked on Google Chrome
+- Open the Extension Management page by navigating to chrome://extensions.
+- The Extension Management page can also be opened by clicking on the Chrome menu, hovering over More Tools then selecting Extensions.
+- Enable Developer Mode by clicking the toggle switch next to Developer mode.
+- Click the LOAD UNPACKED button and select the extension directory(`browser-extension`).
+Ta-da! The extension has been successfully installed
+
+==== Loading unpacked on Mozilla Firefox
+- Open the `about:debugging` page
+- Click "This Firefox" (in newer versions of Firefox)
+- Click "Load Temporary Add-on"
+- Select any file(like `manifest.json`) in the extension's directory(`browser-extension`).
+
+__**P.S: Browser Extensions will soon be published on Browser's App Store to simplify installation**__ .
+
+== Usage
+ 
+=== As a CLI Tool
+- Launch the terminal and type `starfix https://github.com/user_name/repo_name.git` (You can also use ssh)
+- The repository will be cloned in preferred directory and launched in preferred Editor.See the Demo below:
+image:https://user-images.githubusercontent.com/31308705/81454321-8aa82f80-91a9-11ea-9aa2-101c21ced321.gif[Starfix CLI,title="Starfix CLI"]
```

Comment:
please add images to the repository, like in images/*

Reply:
resolved @https://github.com/starfixdev/starfix/pull/29/commits/102717f142cdff59a6a9ce92971c3e971c34342e